content/en/docs/collector/management.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Management
+weight: 23
+---
+
+Please be sure to review the following documentation:
+
+*  [Getting Started][otel-collector-getting-started] to understand how to
+   install the OpenTelemetry collector.
+*  [Configuration][otel-collector-configuration] for how to configure the 
+   OpenTelemetry collector, setting up telemetry pipelines.
+
+## Basics
+
+Telemetry collection at scale requires us to come up with a structured approach
+to manage agents. Typical agent management tasks (ordered from most common to
+least common) include:
+
+1. Querying the agent information and configuration. The agent information can
+include its version, operating system related information, or capabilities.
+The configuration of the agent refers to its telemetry collection setup,
+for example, the OpenTelemetry collector [configuration][otel-collector-configuration].
+1. Upgrading and downgrading agents and management of agent-specific packages,
+including the base agent functionality and plugins.
```

Comment:
```suggestion
1. Upgrading/downgrading agents and management of agent-specific packages,
including the base agent functionality and plugins.
```

"the base agent functionality" --> is that mean to be a part of "agent-specific packages"? If so, what's it referring to in particular? Maybe some examples would be helpful. Although IIRC you can't just manage packages in the collector, but need to build a version of the collector with components baked in.